providers/fips/self_test.c

```diff
@@ -172,6 +172,64 @@ DEP_FINI_ATTRIBUTE void cleanup(void)
 }
 #endif
 
+/*
+ * We need an explicit HMAC-SHA-256 KAT even though it is also
+ * checked as part of the KDF KATs.  Refer IG 10.3.
+ */
+static const unsigned char hmac_kat_pt[] = {
+    0xdd, 0x0c, 0x30, 0x33, 0x35, 0xf9, 0xe4, 0x2e,
+    0xc2, 0xef, 0xcc, 0xbf, 0x07, 0x95, 0xee, 0xa2
+};
+static const unsigned char hmac_kat_key[] = {
```

Comment:
Maybe plonk this in the .inc file?

Reply:
Wrong C file.  This one doesn't include the .inc file.  I think the HMAC needs to have a KAT before the integrity check is done, so here is the spot.